.travis.yml

```diff
@@ -8,7 +8,7 @@ env:
     - ENV=pip
     - PIP_ARGS="--upgrade --use-feature=2020-resolver -e"
     - PIP_SELECTOR="all, tests, coverage"
-    - PYTEST_ARGS="--pyargs hyperspy"
+    - PYTEST_ARGS="--pyargs hyperspy --reruns 3"
```

Comment:
Add `-n 2` to overwrite the setting set in `setup.cfg`?

Reply:
Let's try it and see!

Reply:
Looks promising https://travis-ci.org/github/hyperspy/hyperspy/jobs/723489054

Reply:
Good suggestion!